connexion/apis/flask_api.py

```diff
@@ -35,9 +42,13 @@ def add_openapi_json(self):
         logger.debug('Adding spec json: %s/%s', self.base_path,
                      self.options.openapi_spec_path)
         endpoint_name = "{name}_openapi_json".format(name=self.blueprint.name)
+
+        def get_json_spec():
```

Comment:
Looks like to me that this fits in the class `InternalHandlers`, located in line 300 of this file.